client/src/components/login/login.js

```diff
@@ -85,7 +85,6 @@ class Login extends Component {
   }
 
   handleGoogleOAuthError(response) {
```

Comment:
This was a error message brother! @fave77 we can have toast message for this?

Reply:
@apurv-1 sure